File: ABC/ABC053/C/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  long x;
  cin >> x;

  long count = 2 * (x / 11);

  long mod = x % 11;

  if (0 < mod && mod < 7){
    count += 1;
  } else if (6 < mod) {
    count += 2;
  }

  cout << count << endl;

}

File: ABC/ABC053/C/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    
    long x = sc.nextLong();

    sc.close();

    long count = x / 11 * 2;

    long mod = x % 11;

    if (0 < mod && mod <= 6) {
      count++;
    } else if (mod != 0) {
      count += 2;
    }

    System.out.println(count);
    
  }

}
